frontend/src/model/beta/settings/ModelDetails.tsx

```diff
@@ -0,0 +1,154 @@
+import { Lock, LockOpen } from '@mui/icons-material'
+import {
+  Box,
+  Button,
+  Divider,
+  FormControl,
+  FormControlLabel,
+  Radio,
+  RadioGroup,
+  Stack,
+  TextField,
+  Tooltip,
+  Typography,
+} from '@mui/material'
+import { useTheme } from '@mui/material/styles'
+import _ from 'lodash-es'
```

Comment:
I think this breaks tree-shaking.  Explicitly import the things you want.  Also I think this is actually unused??